.gitattributes

```diff
@@ -25,7 +25,7 @@
 *.coffee text
 *.config text
 *.cs text diff=csharp 
-*.csproj text merge=union 
```

Comment:
@Yuri05 @georgeDaskalakis @abdelr I am trying this to see if this weird behavior with merge is going to disappear